openshift-templates/hawkular-apm-server-deployment.yml

```diff
@@ -39,46 +39,97 @@ items:
             value: admin
           - name: HAWKULAR_APM_ADMIN_PASSWORD
             value: password
+          - name: HAWKULAR_APM_ELASTICSEARCH_HOSTS
+            value: hawkular-apm-es:9300
           image: jboss/hawkular-apm-server
           imagePullPolicy: IfNotPresent
           name: hawkular-apm
-          resources:
-            limits:
-              memory: 2048Mi
-          securityContext:
-            capabilities: {}
-            privileged: false
           terminationMessagePath: /dev/termination-log
           readinessProbe:
-            timeoutSeconds: 5
-            initialDelaySeconds: 20
+            timeoutSeconds: 10
+            initialDelaySeconds: 40
             httpGet:
               path: "/hawkular/apm/status"
               port: 8080
           livenessProbe:
-            timeoutSeconds: 5
-            initialDelaySeconds: 20
+            timeoutSeconds: 10
+            initialDelaySeconds: 40
             httpGet:
               path: "/hawkular/apm/status"
               port: 8080
+          resources:
+            requests:
+              memory: 1024Mi
+              cpu: 2000m
         dnsPolicy: ClusterFirst
         restartPolicy: Always
         serviceAccountName: hawkular-apm
     triggers:
     - type: ConfigChange
 - apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    creationTimestamp: null
+    name: hawkular-apm-es
+  spec:
+    replicas: 1
+    selector:
+      name: hawkular-apm-es
+    strategy:
+      type: Recreate
+    template:
+      metadata:
+        creationTimestamp: null
+        labels:
+          name: hawkular-apm-es
+      spec:
+        containers:
+        - capabilities: {}
+          image: jpkroehling/elasticsearch
```

Comment:
Understand - although would it be better to just create it under jboss-dockerfiles/hawkular-apm, so atleast it is hawkular-apm related rather than your personal account? We can have a readme with it to make clear that this is a community image, as you said?